components/sections/changelog/Changelog.tsx

```diff
@@ -50,7 +52,9 @@ const Changelog: FC<ChangelogProps> = ({
           <IoMdGitCommit className="absolute -left-5 bg-darkBG text-4xl" />
         </span>
         <Typography alignLarge="left" variant="title3">
-          {title}
+          <a href={`/changelog/${slug?.current}`} className="hover:text-brandOrange hover:underline hover:decoration-brandOrange">
```

Comment:
If `slug` is `undefined`, this will result in this url `/changelog/undefined`. If there is no `slug` defined, should this render differently or not at all?

I haven't worked in this project yet. 😅 

![CleanShot 2024-02-01 at 13 19 51](https://github.com/open-sauced/landing-page/assets/833231/c878bcce-f740-434e-bd81-4c2c7eaa7471)

Reply:
I didn't get to that page. I was just console logging that scenario.

Instead of a redirect, wouldn't not rendering that particular changelog make sense? Again, I don't have the full context of this app, so maybe I'm missing something.

Reply:
I think the only ways to really get to these pages are from the `/changelog` page or an external link, eg social media. There's a chance in the future that links will get broken (whether due to mis-spelling, changing slugs in the CMS, or removal).

Here are three options to handle a possible undefined changelog (imo):
1. redirect to a general 404 page that has a link back to the homepage
2. redirect to `/changelog` with an error message that says we can't find that particular log (maybe a dismissible banner?)
3. render the changelog page with an error message and a link back to `/changelog`

Reply:
I think the 404 probably makes sense